test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/ViewFeatures/CookieTempDataProviderTest.cs

```diff
@@ -18,6 +18,37 @@ namespace Microsoft.AspNetCore.Mvc.ViewFeatures
     public class CookieTempDataProviderTest
     {
         [Fact]
+        public void SaveTempData_UsesCookieName_FromOptions()
+        {
+            // Arrange
+            var exepectedCookieName = "TestCookieName";
+            var values = new Dictionary<string, object>();
+            values.Add("int", 10);
+            var tempDataProviderStore = new TempDataSerializer();
+            var expectedDataToProtect = tempDataProviderStore.Serialize(values);
+            var expectedDataInCookie = Base64UrlTextEncoder.Encode(expectedDataToProtect);
+            var dataProtector = new PassThroughDataProtector();
+            var tempDataProvider = GetProvider(dataProtector, new CookieTempDataProviderOptions() { CookieName = exepectedCookieName });
+            var responseCookies = new MockResponseCookieCollection();
+            var httpContext = new Mock<HttpContext>();
+            httpContext
+                .SetupGet(hc => hc.Request.PathBase)
+                .Returns("/");
+            httpContext
+                .Setup(hc => hc.Response.Cookies)
+                .Returns(responseCookies);
```

Comment:
Just a suggestion, some whitespace to break up these lines into logical blocks would go a long way for improving readability. This is a wall of text 😆